pkg/lang/python/plugin_persist.go

```diff
@@ -515,11 +511,15 @@ func (p *persister) queryFS(file *core.SourceFile, annotation *core.Annotation,
 	module, aliasedModule, alias, importStatement := match["module"], match["aliasedModule"], match["alias"], match["importStatement"]
 
 	// this assignment/invocation is unrelated to aiofile instantiation found from the matching import
+	var varName string
 	if aliasedModule != nil {
-		if !query.NodeContentEquals(alias, varName) {
-			return nil
+		if content, matched := query.GetMatchingNodeContent(alias, varNames); matched {
```

Comment:
nit: (opinion) I don't feel like this is easier to read than `_, matched := varNames[alias.Content()]` (and using `alias.Content()` inside). Especially not clear that the first argument is just the `.Content()` of the node passed in.

Reply:
Is that true of the `query.NodeContentIn` call? If you like that one but not this one, I could also just do:

```go
if query.NodeContentIn(alias, varNames) {
  content := alias.Content() // an extra call to Content(), but that's probably fine
  ...
```

Reply:
☝️ I went with that